lib/tasks/marks.rake

```diff
@@ -4,12 +4,11 @@ namespace :db do
   task :marks => :environment do
     puts 'Assign Marks for Assignments'
 
-    #Function used to create marks for both criterias
-    def create_mark(result_id, markable_type, markable)
+    # Creates marks for all criteria
+    def create_mark(result_id, markable)
       Mark.create(
         result_id: result_id,
-        mark: rand(0..4),
-        markable_type: markable_type,
+        mark: rand(0..markable.max_mark).floor,
```

Comment:
I changed this. I have not uploaded my latest work because I thought there was a bug, but the bug seems to be in master. When I run db:reset in master and check assignment 3 or 4, I see a red table.